### PR TITLE
Issue 461 сreate user profile page

### PR DESCRIPTION
**Pull request by Valentyna-Chudik**
develop

## GitHub Board

**Issue link**

[#461 Create user profile page](https://github.com/ita-social-projects/dokazovi-fe/issues/461 )

## Summary of change

- [x]  Create user profile page

### CODE REVIEW

**Review by AndriiKachkarov — APPROVED** [review]

**Review by supersoild2000 — APPROVED** [review]
Great!

**Review by supersoild2000 — APPROVED** [review]

File: src/components/Header/AccountMenu.tsx [hunk 1]
```diff
@@ -37,6 +40,11 @@ export const AccountMenu: React.FC = () => {
     setAnchorEl(null);
   };
 
+  const openProfile = () => {
+    history.push('/profile');
```
Comment (michalenr) [inline src/components/Header/AccountMenu.tsx:44]: why not use React router Link?

File: src/views/Profile/OperationView.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+interface IOperationViewProps {
+  selectedOption: IProfileMenuOption | Record<string, never>;
+}
+
+export const OperationView: React.FC<IOperationViewProps> = (props) => {
+  const { selectedOption } = props;
+  const { value } = selectedOption;
+  const classes = useStyles();
+
+  const renderOperationView = () => {
```
Comment (michalenr) [inline src/views/Profile/OperationView.tsx:16]: It  would be cleaner to use dispatching here

```jsx
  const operationViews = {
    info: <div>Info</div>,
    operationView: <MaterialsView />,
    passwordChange: <div>Password Change</div>,
    mail: <div>Mail</div>,
  };
  return (
    <Container className={classes.operationView}>
      {operationViews[value] ?? <div>Info</div>}
    </Container>
  );

```

File: src/components/Header/AccountMenu.tsx (hunk 1 above)
Comment (Valentyna-Chudik) [reply src/components/Header/AccountMenu.tsx]: Done (React router Link used)

File: src/views/Profile/OperationView.tsx (hunk 2 above)
Comment (Valentyna-Chudik) [reply src/views/Profile/OperationView.tsx]: Done (yes, it's much more better)

**Review by supersoild2000 — APPROVED** [review]

**Review by supersoild2000 — APPROVED** [review]

**Review by michalenr — APPROVED** [review]

**Pull request merged**